### DISCUSSION TITLE
Scripting Symfony commands with arguments from the Linux commandline

### DISCUSSION THREAD

**Discussion by hg2581** [symfony/symfony#51885] [Q&A] [answered]
I am new to Symfony and will be using 6.2 for a project. I would like to be able to script a series of Symfony commands to run in a bash script from the command line. There does not seem to be any issue scripting various console commands:

symfony console make:entity mytest

but, how can I include all the different properties/fields to the above? An example could be:

amount, float, yes, total, integer, no

etc etc.

What would the appropriate syntax be for including any arguments to a symfony command like the one above?

**Comment by tacman** [comment]
Completely 100% non-standard hack, but this is what i do when I need a script that will create an entity from scratch:

```bash
cat <<'EOF' | sed "s/:/\n/g"  |  bin/console make:entity Country
name:string:55:no
alpha2:string:2:no
alpha3:string:3:yes
EOF
```

**Comment by hg2581** [comment]
Interesting, I will try that! I would have expected it to be a "standard" way of doing this... You and I are likely not the first to want to do this, I would think...

**Reply by tacman** [reply, marked as answer]
There's an issue in the maker bundle somewhere on this exact issue, and the decision was to not support this because of the backwards compatibility promise.  That makes sense.

I'd like to rewrite an entity maker that included an option for each property to belong to a serializer group.  And someone else might want to add validation via an attribute.  Or security or who knows what else.  So I can understand the decision to keep it as it.

**Reply by hg2581** [reply]
Not sure what this involves but I think scripting arguments would be a very useful extension of capabilities!
